tests/test_dt_expr.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 import random
 import datatable as dt
-from datatable import f, stype, ltype
+from datatable import f, stype, ltype, first, count
```

Comment:
looks like you don't need to import first and count here anymore

Reply:
Sure, removed.